.github/release.yml

```
@@ -0,0 +1,15 @@
+changelog:
+  categories:
+    - title: Testable features, fixes, enhancements 🎉
+      labels:
+        - '*'
+      exclude:
+        labels:
+        - not-testable
+        - Nuernberg
+    - title: Testable features for nuernberg 🎉
```

:upside_down_face: 
```suggestion
    - title: Nuernberg :tada: 
```
Maybe we even have a more distinct emoji on top? For example :orange_heart: ?

---

I think both are features that have to be tested. I don't we have to use different emojis tbh

---

> I think both are features that have to be tested

I think thats clear without putting `Testable` in the category name.

---

ok. md emoji usage won't work here.